<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/Calculator_30001.png</normaloff>../../Downloads/Calculator_30001.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Calculo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>39</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 87 14pt &quot;Arial Black&quot;;color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>39</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_salir">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 127);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_parentesis1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>81</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
    <property name="shortcut">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_parentesis2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>124</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
    <property name="shortcut">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_punto">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>167</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="shortcut">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="igual">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(206, 206, 206);
font: 87 14pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="shortcut">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="Calculo_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 87 10pt &quot;Arial Black&quot;;color:rgb(202, 202, 202)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>80</y>
      <width>247</width>
      <height>287</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="boton1">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Calculadora</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="shortcut">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="boton2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="shortcut">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="boton3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="shortcut">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="boton4">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="shortcut">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="boton5">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="shortcut">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="boton6">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <property name="shortcut">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="boton7">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <property name="shortcut">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="boton8">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="shortcut">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="boton9">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
         <property name="shortcut">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="suma">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="shortcut">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="boton0">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="shortcut">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="boton_borra1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 169, 98);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="shortcut">
          <string>Esc</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="boton_por">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>x</string>
         </property>
         <property name="shortcut">
          <string>*</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="boton_division">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>÷</string>
         </property>
         <property name="shortcut">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="boton_borratodo">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 169, 98);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>AC</string>
         </property>
         <property name="shortcut">
          <string>Backspace</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="boton_resta">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="shortcut">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="boton_resto">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="boton_raiz">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>√</string>
         </property>
         <property name="shortcut">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="potencia">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(137, 255, 155);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string> ^</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>boton_salir</zorder>
   <zorder>boton_parentesis1</zorder>
   <zorder>boton_parentesis2</zorder>
   <zorder>boton_punto</zorder>
   <zorder>igual</zorder>
   <zorder>label_2</zorder>
   <zorder>Calculo</zorder>
   <zorder>Calculo_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_salir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_borratodo</sender>
   <signal>clicked()</signal>
   <receiver>Calculo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_borratodo</sender>
   <signal>clicked()</signal>
   <receiver>Calculo_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
